FaunaDB.Client.Test/ClientTest.cs

```diff
@@ -1501,7 +1501,8 @@ public async Task TestEchoQuery()
             Assert.That(((QueryV)result).Value, Is.EquivalentTo(new Dictionary<string, Expr>
             {
                 {"lambda", Arr("x", "y")},
-                {"expr", Concat(Arr(Var("x"), "/", Var("y")))}
+                {"expr", Concat(Arr(Var("x"), "/", Var("y")))},
+                { "api_version", "3" }
```

Comment:
This should be an integration test, right? :)

Reply:
This class is testing the "whole client", so it can be seen as a integration test (with lots of stuff on it). We definitely ~could~ need to improve those tests and refactor'em out in smaller classes.